report/completeness.html

```diff
@@ -0,0 +1,147 @@
+<html>
+
+<head>
+    <title>Mapaction Completeness Report</title>
+    <link rel="icon" href="favicon.png" type="image/x-icon">
+    <!-- Note fot Mapction. You can change favicon with yours -->
+    <link rel="stylesheet" href='completeness.css'>
+    <link href='https://fonts.googleapis.com/css?family=IBM Plex Sans Condensed' rel='stylesheet'>
+    <script src="countryList.js" type="text/javascript"> </script>
+</head>
+
+<body>
+    <div id="header"></div>
+    <div id="completenessTable"></div>
+</body>
+
+<script>
+    const selectCountry = `Please select a country`
+    //Note for Mapaction. You can change logo with yours.
+    let headers = `
+            <div id="compReport">
+                <img class="img" src='Kontur_logo_main.png' />
+             <div class="vertical-center">
+            <p>COMPLETENESS REPORT</p>
+            </div>
+            </div>
+                <h1 id="countryHeader">${selectCountry}</h2>
+                    <div>
+                        <input autocomplete="false" type="text" id="countryInput" name="search" onkeyup="filterCountriesFnc()" placeholder="Search for country .." />
+                        </div>
+                    <div>
+                    <select id="countrySelect" size="3" onchange="countrySelectFnc()">`;
+    Object.keys(countries).forEach(function (key) {
+        headers += `<option value=${key}>${countries[key]}</option>`;
+    })
+    headers += "</select></div>";
+
+    document.getElementById("header").innerHTML = headers;
+
+    function formatDate(date) {
+        if (date == null || date == "") {
+            return "";
```

Comment:
should we better return "no data" text instead of white space?

Reply:
if no data, no file name and no date is returned.
Relevant line is shown red.
But returned null insted of ""